<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>634</width>
    <height>368</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="licenseTableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>611</width>
     <height>181</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="licenseTableWidgetLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Luvat</string>
   </property>
  </widget>
  <widget class="QPushButton" name="populateFromTablePushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Täytä</string>
   </property>
  </widget>
  <widget class="QPushButton" name="savePushButton">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>320</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Tallenna</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelPushButton">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>320</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Peruuta</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="licenseYearLineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="licenseYearLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Lupavuosi</string>
   </property>
  </widget>
  <widget class="QLabel" name="animalLabel">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>260</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Eläin</string>
   </property>
  </widget>
  <widget class="QLabel" name="animalAgeGroupLabel">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>260</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Ikäryhmä</string>
   </property>
  </widget>
  <widget class="QLabel" name="animalGenderLabel">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>260</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Sukupuoli</string>
   </property>
  </widget>
  <widget class="QLabel" name="licenseAmountLabel">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>260</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Määrä</string>
   </property>
  </widget>
  <widget class="QComboBox" name="animalComboBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>280</y>
     <width>86</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="animalGenderComboBox">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>280</y>
     <width>86</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="animalAgeGroupComboBox">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>280</y>
     <width>86</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="licenseAmountLineEdit">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>280</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
